### PR TITLE
BrickHack 8 - Adding Discord to site

### PR DESCRIPTION
**Pull request by sjmiller7**
to merge when discord is ready

### CODE REVIEW

**Review by peterkos — APPROVED** [review]
LGTM! needs backmerge tho

File: sass/index.scss [hunk 1]
```diff
@@ -266,7 +266,7 @@ video::-webkit-media-controls {
             justify-content: space-between;
 
             #login {
-                @include button-primary;
+                @include button-secondary();
```
Comment (skyegallup) [inline sass/index.scss:269]: Does this need parentheses, or can they be removed?

File: sass/index.scss [hunk 2]
```diff
@@ -276,6 +276,12 @@ video::-webkit-media-controls {
                 width: 50%;
             }
         }
+        
+        #discord {
+            @include button-primary();
```
Comment (skyegallup) [inline sass/index.scss:281]: Same parentheses comment

**Review by skyegallup — APPROVED** [review]
One style nit/question, but otherwise 👍

File: sass/index.scss (hunk 1 above)
Comment (sjmiller7) [reply sass/index.scss:269]: doesn't need them but for the sake of not dealing with a review  im leaving them

**Pull request merged**